WordPress/gutenberg
# Fix: ZoomOut scrollbar appearing at middle issue

## What?
removed unnecessary CSS causing issues with the scrollbar to appear in the middle.
## Why?
fixes https://github.com/WordPress/gutenberg/issues/65080


## Testing Instructions
- open site editor
- open blocks sidebar
- open styles & then click on `Browse styles`
- close blocks sidebar
- The scrollbar will appear at the end.

## Screenshots or screencast 

https://github.com/user-attachments/assets/784f44b0-dc91-4c73-a2d6-403c6096a2ca
